### DISCUSSION TITLE
Please Add Power Consumption Stats!

### DISCUSSION THREAD

**Discussion by itistripleR** [angelbbs/NiceHashMinerLegacy#132] [Ideas]
Hai, 

I'm just happy that using This Forkfix, Great Job Guys, this Helps More for me!

But if you can do some tweaks, this app will be the BEST!

1 - Whenever I change the settings, I need to restart Mining for changes tobe applied, also if I disabled one Gpu, it should instantly works without affecting another gpu's from mining, please Fix!

2 - this is Important Reason for Write this Comment,

Please add some Power Consumption Statistics with total System Uptime and how many Power it Consumed since Power On (For Ref:- Gminer Statistics)

3 - I think its bug, Sometimes When I restart the PC, the Program Resets Automatically, Starting from Fresh, don't know why its happening,

So These three Suggestions only From My Side, Hope You understand my bad Grammer,

Cheers, Thanks!

**Comment by angelbbs** [comment]
1. Restarting the miner is a prerequisite for accepting new settings
2. In the following releases
3. Go "Settings->About->Backup-Create" partially solves this problem
